Documentation/PageTsconfig/TceForm.rst

```diff
@@ -63,1113 +65,1206 @@ dots ('.'), these must be escaped with backslash.
 
-      TCEFORM.pages.doktype {
-         # Different "INVALID VALUE ..." label:
-         noMatchingValue_label = VALUE "%s" was not available!
-      }
+    TCEFORM.pages.doktype {
+        # Different "INVALID VALUE ..." label:
+        noMatchingValue_label = VALUE "%s" was not available!
+    }
 
-   .. figure:: /Images/ManualScreenshots/List/SelectInvalidValueDifferentLabel.png
-      :alt:  An invalid selector box value is indicated by a warning message
+..  figure:: /Images/ManualScreenshots/List/SelectInvalidValueDifferentLabel.png
+    :alt:  An invalid selector box value is indicated by a warning message
 
-      An invalid selector box value is indicated by a warning message
+    An invalid selector box value is indicated by a warning message
 
 
-.. index::
-   Records; select items remove
+..  index::
+    Records; select items remove
+..  _tceform-removeItems:
 
 removeItems
 -----------
 
-:aspect:`Datatype`
-   list of values
+..  confval:: removeItems
+    :name: tceform-removeItems
+    :type: list of values
 
-:aspect:`Description`
-   Change the list of items in :ref:`TCA type=select <t3tca:columns-select>` fields. Using this property,
-   single items can be removed, leaving all others.
+    Change the list of items in :ref:`TCA type=select <t3tca:columns-select>` fields. Using this property,
+    single items can be removed, leaving all others.
 
-   This property is available for various levels:
+    This property is available for various levels:
 
-   table level, example:
-      `TCEFORM.tt_content.header_layout.removeItems`
+    table level, example:
+        `TCEFORM.tt_content.header_layout.removeItems`
+
+    table and record type level, example:
+        `TCEFORM.tt_content.header_layout.types.textpic.removeItems`
 
-   table and record type level, example:
-      `TCEFORM.tt_content.header_layout.types.textpic.removeItems`
+    Flex form field level, example:
+        `TCEFORM.tt_content.pi_flexform.myext_pi1.sDEF.settings\.myfield.removeItems`
 
-   Flex form field level, example:
-      `TCEFORM.tt_content.pi_flexform.myext_pi1.sDEF.settings\.myfield.removeItems`
+        Where `sDEF` is the sheet name. For a description
+        see the section :ref:`tceformApplyPropertiesFlexForm` on this page.
 
-      Where `sDEF` is the sheet name. For a description
-      see the section :ref:`tceformApplyPropertiesFlexForm` on this page.
+Example: Remove "Recycler" and "Spacer" page types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:aspect:`Example`
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/page.tsconfig
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/page.tsconfig
 
-      TCEFORM.pages.doktype {
-         # Remove "Recycler" and "Spacer" page types
-         removeItems = 199, 255
-      }
+    TCEFORM.pages.doktype {
+        # Remove "Recycler" and "Spacer" page types
+        removeItems = 199, 255
```

Comment:
Just a note (no action needed): Recycler is not available in v13 anymore.

Reply:
I did not make content changes here, as I wanted to avoid (unsucsessfully) the PR becomming to big.